### PR TITLE
Add example descriptions, data links, and image links to dialog

### LINKED ISSUE #32: Add descriptions for examples
In place of definitions, say something about where they are located

### PR DESCRIPTION
**Pull request by hcorson-dosch**
This PR modified the dialog content to:

* Add descriptions for the example volumes - closes #32
* Precede the definition/description with the text 'Definition: ' or 'Description: '
* Make the image in the dialog clickable - link to image website
* Make the pool volume/flux rate a link to the data source, including for example volumes
* Adjust the dialog size so that it doesn't extend off the page on a short & wide laptop-sized window

### CODE REVIEW

**Comment by ceenell** [comment]
The space is a little awkward with the image and the text below it on a wider view. I think setting a max-width might make sense here.
image

**Comment by hcorson-dosch** [comment]
> The space is a little awkward with the image and the text below it on a wider view. I think setting a max-width might make sense here. image

There is a `max-width` of 60% at the moment. Agreed it doesn't look great, but it's tricky b/c it's a balancing act  to keep it functional on a short wide screen like a laptop. If I lower it, the 'close' button gets cut off for items w/ longer definitions:
image

I've switched instead to wrapping the definition text around the image:
image
image

**Comment by ceenell** [comment]
Sweet, that looks great. So on mobile it ends up looking more stacked? I think this is a good solution and I'm happy with it, but just to respond to the max-width thing, in this type of situation a specific pixel value would probably work. That same idea can be applied to text, where a certain specified dimension is more readable than a relative value.

**Review by ceenell — APPROVED** [review]

**Comment by hcorson-dosch** [comment]
> Sweet, that looks great. So on mobile it ends up looking more stacked? I think this is a good solution and I'm happy with it, but just to respond to the max-width thing, in this type of situation a specific pixel value would probably work. That same idea can be applied to text, where a certain specified dimension is more readable than a relative value.

hah! of course. Sorry - my brain is a bit fried at the end of each day 🤪

Looks like this on mobile:
image

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # The volume of water on earth
-An interactive logpile chart of major pools and fluxes of water inspired by Duncan Geere's [Carbon in Context](https://www.duncangeere.com/carbonincontext/) chart. Data for water cycle components come from Abbott 2019. 
+An interactive chart of major pools and fluxes of water. Data for water cycle components come from [Abbott et al., 2019](https://www.nature.com/articles/s41561-019-0374-y).
```
[the remark this answers is not in the document]
Comment (hcorson-dosch) [reply README.md:2]: Just pushed a commit to drop the data link in the dialog boxes for all Abbott pools and fluxes.

**Pull request merged**